apps/web/components/BasePrice/types.ts

```diff
@@ -0,0 +1,7 @@
+export type BasePriceProps = {
+  unitContent?: number;
+  unitName?: string;
+  basePrice?: string;
+  oneline: boolean;
+  contentLineFirst: boolean;
```

Comment:
**question**: Do we actually need `contentLineFirst`? In all use cases so far, it gets passed as `true`.

Reply:
<img width="1122" alt="Screenshot 2023-08-23 at 11 13 12" src="https://github.com/plentymarkets/storefront-nuxt3-boilerplate/assets/121931658/a4be089c-5943-4e21-8e43-a78ed502cae4">

Reply:
if you will have cart sidebar and want to respect initial design you just switch it to false
https://github.com/plentymarkets/plentyshop-pwa/pull/238/files#diff-5d1c1af86406377af1bd6219832c64ac513e996076afd808bce19a4867bc0983

Reply:
Tim said i should have functionality exactly as in pwa